### ISSUE TITLE
Use has to click the CONTINUE button for the wallet to prompt me to approve my address on the ledger

### ISSUE THREAD

**Issue by timstackblock** [blockstack/stacks-wallet#233]
**Describe the bug**
There is a weird mis-order in the last step of the ledger process. I have to actually click the blue continue button to get my ledger to prompt me to confirm my address on the device. 

This is a really confusing issue that will send many people to support because they will get stuck at the last step.

**To Reproduce**
1. Go to I ALREADY HAVE A WALLET
2. Click on CONTINUE WITH LEDGER
3. Go through the steps to connect the wallet you are never prompted to verify your address user just waits
4. Once I click continue then I am prompted to verify my address.  But how would a user know this?

**Expected behavior**
The user is waiting to be prompted to verify their address but it doesn't happen until they click continue

**Screenshots**
<img width="1009" alt="Screen Shot 2020-09-11 at 2 33 42 PM" src="https://user-images.githubusercontent.com/49165468/92960559-d3b00f00-f43b-11ea-9a1f-478ed5592049.png">

**Comment by jasperjansz** [comment]
https://www.figma.com/file/VcXbxT1EwXvE1nUlOiBEES/Wallet?node-id=3610%3A26658

**Comment by markmhx** [comment]
@jasperjansz Is the idea here that the button label starts as "Continue" and changes to "Send confirmation to Ledger" once reaching step 3?

**Comment by jasperjansz** [comment]
@markmhx No, there's no button until you get to step 3. The button is only for step 3, to send the request to your Ledger. You can not confirm on your Ledger without clicking the button.

**Comment by GinaAbrams** [comment]
Have seen a couple users run into this since (and it stumped me too!), are we able to provide an upgrade or fix here? 🙏

**Comment by markmhx** [comment]
@kyranjamie does your recent Ledger refactor work resolve this interaction implicitly by chance?

**Comment by kyranjamie** [comment]
Yes. Ledger changes should fix all issues (and hopefully add no new ones 😅)

**Comment by markmhx** [comment]
Great! I'm moving to the Kanban board then so we can QA when time is ready.

**Comment by markmhx** [comment]
I'm still seeing a "Continue" button here in v4.1.1 that needs clicking before the Ledger device shows an address for confirmation per the third step.

Did you intend to remove this button and have the wallet automatically prompt for the address on the device without it? That's the UX change sought here (since some users don't realize they have to click the continue button to trigger the address confirmation prompt).

**Comment by kyranjamie** [comment]
No, I'd never intended to do this. Despite my earlier comment suggesting otherwise, I must've though this was a separate issue. 

Agree we should do this -- though the button does animate from disabled to active when the second step is complete.

So if it needs more spelling out, we should add a Ledger Live-like illustration in addition to just auto-requesting from the device. Personally, I find it much easier to miss something changing (that requires action) on the Ledger itself.

![image](https://user-images.githubusercontent.com/1618764/112002822-96ca8080-8b20-11eb-9200-237132868832.png)

I've put back to backlog as this was never attempted.

**Comment by markmhx** [comment]
> or implement Jasper's suggestion of only displaying button when last step is complete. Which is a much easier change (with fewer edge cases).

Let's do this to keep things simple. We can tackle the addition of an animation in a separate issue.

Let's update the copy a bit as well so the user knows better where to look exactly.

1. Connect your Ledger device to your computer
2. Open the Stacks app on your Ledger device
3.  Confirm your address on your Ledger device
